glossary.asciidoc

```diff
@@ -504,19 +499,18 @@ upstream payment::
     TBD.
 
 wallet::
-    Software that holds all your Bitcoin addresses and secret keys. Use it to send, receive, and store your bitcoin.
+    A walledt is a piece of software that holds Bitcoin addresses and secret keys. It is used to send, receive, and store bitcoin.
     
 watchtower::
     Watchtowers are a security service on the Lightning network that monitor channels.
-    In the case that one of the channel partners goes offline or loses their backup, a watchtower keep their own backups and can restore their channel information.
-    They also monitor the Bitcoin blockchain and can submit a penalty transaction in the case that one of the partners tries to "cheat" by broadcasting an outdated state.
+    In case one of the channel partners goes offline or loses their backup, a watchtower keeps its own backups and can restore channel information.
```

Comment:
I think the old sentence was more correct

Reply:
The original appears grammatically incorrect.  Surely it must be either:

a watchtower keeps its  (singular)
watchtowers keep their (plural)

Reply:
Now that I read the sentence again, I think both are correct, it just depends on how you interpret them. I could be wrong, but as far as I've understood, watchtowers keep backups of channel information of the two channel partners. Do you say that the watchtower keeps ITS own backup (intended as the backup kept by the watchtower) or do you say that the watchtower keeps THEIR backup (intended as the backup of the two channel partners) ?

Reply:
Unfortunately, I don't know enough about how the technology works to make a judgement on that.

Reply:
I think `keeps its` can be used. as far as the backup / copy of channel state it's the watchtowers backup

Reply:
Agreed.  Apologies for the slow response.